src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -946,6 +945,15 @@ protected double getDenominator()
         rowIndexSize = createTableHistogram("RowIndexSize", cfs.keyspace.metric.rowIndexSize, false);
     }
 
+    private Memtable.MemoryUsage getMemoryUsageWithIndexes(ColumnFamilyStore cfs)
+    {
+        Memtable.MemoryUsage usage = Memtable.newMemoryUsage();
+        cfs.getTracker().getView().getCurrentMemtable().addMemoryUsageTo(usage);
```

Comment:
nit: Could we just use `Memtable.MemoryUsage usage = Memtable.getMemoryUsage(cfs.getTracker().getView().getCurrentMemtable())` here?

Reply:
> Memtable.getMemoryUsage could be doing something further like caching the object.

Fair enough, although that would seem more likely if `Memtable.getMemoryUsage()` wasn't a static utility.